Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>target/\r\npom.xml.tag\r\npom.xml.releaseBackup\r\npom.xml.versionsBackup\r\npom.xml.next\r\nrelease.properties\r\ndependency-reduced-pom.xml\r\nbuildNumber.properties\r\n.mvn/timing.properties\r\n# https://maven.apache.org/wrapper/#usage-without-binary-jar\r\n.mvn/wrapper/maven-wrapper.jar\r\n\r\n# Eclipse m2e generated files\r\n# Eclipse Core\r\n.project\r\n.idea/\r\n# JDT-specific (Eclipse Java Development Tools)\r\n.classpath\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 0bcf4e27e12d3198dabaddcd2da81724a0a06911)
+++ b/.gitignore	(date 1773192516250)
@@ -1,4 +1,5 @@
 target/
+.ai/
 pom.xml.tag
 pom.xml.releaseBackup
 pom.xml.versionsBackup
